### PR TITLE
Add Fackbook Prophet Model Stock Prediction Use-case

### PR DESCRIPTION
**Pull request by PeterQiu0516**
Add Fackbook Prophet Model Stock Prediction Use-case.

### CODE REVIEW

**Comment by shane-huang** [comment]
* change title to Stock Price Prediction with ProphetForecaster and AutoProphet (tuned using AutoTS). 
* for Prophet Model demonstration, we shall use ProphetForecaster instead of the internal ProphetModel. please submit another PR to add ProphetForecaster to inherit from zouwu.forecast.Forecaster. And use ProphetForecaster to do the forecasting.

**Comment by shane-huang** [comment]
Also add the notebook to run test scripts.

**Comment by PeterQiu0516** [comment]
Implement ProphetForecaster in #4134.

File: pyzoo/zoo/chronos/use-case/fsi/README.md [hunk 1]
```diff
@@ -5,15 +5,15 @@ We demonstrate how to use Chronos to predict stock prices based on historical ti
 
 In the reference use case, we use the publicly available daily stock price of S&P500 stocks during 2013-2018 ([data source](https://www.kaggle.com/camnugent/sandp500/))
 
-This use case example contains a notebook:
+This use case example contains 2 notebook:
```
Comment (shane-huang) [inline pyzoo/zoo/chronos/use-case/fsi/README.md:8]: 2 notebooks

**Comment by shane-huang** [comment]
Notebook:  change title from "Stock Price Prediction with ProphetForecaster and AutoProphet (tuned using AutoTS)" to "Stock Price Prediction with ProphetForecaster and AutoProphet (with AutoML)"

**Comment by shane-huang** [comment]
Others generally LGTM.

**Comment by PeterQiu0516** [comment]
Jenkins passed: http://10.239.176.111:18888/job/ZOO-PR-Validation/5929/
http://10.239.176.111:18888/job/ZOO-PR-Python-Spark-2.3-py37/2763/

**Pull request merged**